modules/oadp-ibm-power-test-matrix.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: CONCEPT
+[id="oadp-ibm-power-test-matrix_{context}"]
+= OADP support for target backup locations using IBM Power Systems
+
+The table that follows shows which target backup locations are supported for IBM Power Systems ("cloud systems") running on OCP 4.11 and OADP 1.1.0:
```

Comment:
I think we need to be more accurate on the wording here, and should state that it was tested only on IBM COS and AWS S3 and not adding a compatibly table, as theoretically the object storage is suppose to be platform agnostic.
@shubham-pampattiwar/@dymurray/@sseago could you please confirm (or correct)?